components/Navigator/components/Content/components/Search/style.js

```diff
@@ -26,25 +26,26 @@ const input = css({
 
 const overlay = css({
   position: 'fixed',
-  left: -variables.navigator.height,
   top: variables.navigator.height,
```

Comment:
since the backdrop has a bg color now, it looks broken on iPhone X. We need to consider `--safe-area-inset-top` here. This might also be added to the Backdrop component if we use it here